src/lighteval/utils/imports.py

```diff
@@ -93,3 +94,51 @@ def can_load_extended_tasks() -> bool:
 
 
 CANNOT_USE_EXTENDED_TASKS_MSG = "If you want to use extended_tasks, make sure you installed their dependencies using `pip install -e .[extended_tasks]`."
+
+
+# TODO: Might be nice to merge this with functions above
```

Comment:
Please merge it with the above functions indeed so we don't duplicate code logic